sycl/test-e2e/bindless_images/sampled_fetch/fetch_2D.cpp

```diff
@@ -1,8 +1,7 @@
-// REQUIRES: cuda
-// REQUIRES: aspect-ext_oneapi_bindless_sampled_image_fetch_2d
+// REQUIRES: aspect-ext_oneapi_bindless_images
 
```

Comment:
Similarly to above, why is `// REQUIRES: aspect-ext_oneapi_bindless_sampled_image_fetch_2d` being removed?

The `aspect-ext_oneapi_bindless_images` alone doesn't guarantee that 2D sampled image fetch will work, and it is being used in the kernel.